src/components/molecules/Comment/Comment.jsx

```diff
@@ -3,41 +3,47 @@ import { Text } from '../../atoms/Text/Text';
 import { dateFormat } from '../../../utils/format';
 import '../../atoms/Text/Text.scss';
 import './Comment.scss';
+import { useUserDetails } from '../../../hooks';
+
+export const Comment = ({ authorID, date, children }) => {
+    const author = useUserDetails(authorID);
```

Comment:
const {data} = useUserDetails

Also this low lvl shouldn't be responsible for API call, but for now I think is fine, we have to deliver the project :D
Just be aware of that